js-intro.js

```diff
@@ -4,58 +4,195 @@
 var temp3 = 212
+// Creating boiling point temp
+var tempBoil = 212
+
+//Creating a series of functions for each temp
+
+const isItBoiling1 = (butseriouslyisit) => {
+    if(temp1 < tempBoil ) {
+        return `${temp1} is below the boiling point`
+    }
+}
+
+const isItBoiling2 = (butseriouslyisit) => {
+    if(temp2 > tempBoil ) {
+        return `${temp2} is above the boiling point`
+    }
+}
+
+const isItBoiling3 = (butseriouslyisit) => {
+    if(temp3 || tempBoil ) {
+        return `${temp3} is at the boiling point`
+    }
+}
 
+//Logging the results to console.
+//Expected result ----> 35 is below the boiling point
+350 is above the boiling point
+212 is at the boiling point
+console.log(isItBoiling1())
+console.log(isItBoiling2())
+console.log(isItBoiling3())
 
 
+*/
 
-// --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
-// Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
+//-------------------------------------------------------------
 
+
+// // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
+// // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
+/*
 var myNumbers1 = [3, 7, 0, 6, -9]
 
 
+//Creation of multiply function
+const multiplyBy = (multiplied) => {
+    for(let i=0; i<myNumbers1.length; i++){
+    console.log(myNumbers1[i] * 5);
+    }
+}
+
+// Calling and logging the multiplication 
+multiplyBy()
+
+*/
+//--------------------------------
+
 
+// // --------------------3) Create a function that multiplies each number in the array by 5 using map.
+// // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
-// --------------------3) Create a function that multiplies each number in the array by 5 using map.
-// Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
+/*
+
+//Predefined variable
 var myNumbers2 = [8, -7, 0, 6, 2]
 
+//Creating the new array with multiplication
+const multiplyByArr = myNumbers2.map(function(value){
+    return value * 5
+})
+
+
+//Logging the multiply. 
+console.log(multiplyByArr)
 
+*/
+//--------------------------
 
+// // --------------------4) Create a function that removes all the vowels from a string.
+// // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
-// --------------------4) Create a function that removes all the vowels from a string.
-// Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
+/*
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
 
+//Creation of new function to remove vowels
+const nayVow = defNoVows => {
+  let newArray1 = defNoVows.split("")
+  let newNewArray = newArray1.filter(value => value.toLowerCase() !== "a" && value.toLowerCase() !== "e" && value.toLowerCase() !== "i" && value.toLowerCase() !== "o" && value.toLowerCase() !== "u")
```

Comment:
Instead of `newArray1`, `newNewArray`, etc. consider naming your variables here a little more clearly. Example: `inputSplit`, `inputMapped`, and `finalOutput`.

Reply:
Excellent point, I'll keep this in mind for now on!